### PR TITLE
Wheel examples

### PR DESCRIPTION
**Pull request by titusfortner**
This is Draft because:
* the gh-codeblock is temporarily pointing to the wrong branch for demo purposes & needs to be changed back before merging
* Depends on merging https://github.com/SeleniumHQ/selenium/pull/10650

This PR is to show off demo code for what scrolling implementation looks like with the above PR

Also, probably should figure out if we like the badges and do them the right way with shortcodes and such if we like them. @diemol what do you think of the "badges" in this PR?

### CODE REVIEW

**Comment by titusfortner** [comment]
Oh, also, the JavaScript & the Kotlin aren't implemented for this because I wasn't sure what those needed to look like.

**Comment by titusfortner** [comment]
Do I need to move this out of draft to get it to deploy a preview?

File: examples/java/src/test/java/dev/selenium/actions_api/WheelTest.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+    private boolean inViewport(WebElement element) {
+
+        String script =
+                "for(var e=arguments[0],f=e.offsetTop,t=e.offsetLeft,o=e.offsetWidth,n=e.offsetHeight;\n"
+                        + "e.offsetParent;)f+=(e=e.offsetParent).offsetTop,t+=e.offsetLeft;\n"
+                        + "return f<window.pageYOffset+window.innerHeight&&t<window.pageXOffset+window.innerWidth&&f+n>\n"
+                        + "window.pageYOffset&&t+o>window.pageXOffset";
+
+        return (boolean) ((JavascriptExecutor) driver).executeScript(script, element);
```
Comment (alaahong) [inline examples/java/src/test/java/dev/selenium/actions_api/WheelTest.java:101]: executeScript will return Boolean as Object, seems no need to convert as boolean?

**Comment by diemol** [comment]
I like the badges idea, but there is no need to implement this on our own. Docsy uses Bootstrap, so you can use their Badges (https://getbootstrap.com/docs/4.6/components/badge/) and combine it with a Tooltip (https://getbootstrap.com/docs/4.6/components/tooltips/). 
That could be embedded in a shortcode, definitely.

**Comment by titusfortner** [comment]
Getting rolled into #1031

**Pull request closed without merging**